src/utils/api.ts

```diff
@@ -0,0 +1,104 @@
+const apiUrl = "https://disease.sh/v3/covid-19/";
+
+async function request(endpoint: string) {
+  const response = await fetch(apiUrl + endpoint);
+  return response.json();
+}
+
+export async function getAllCountryData() {
+  const countries = await request(`countries`);
+  countries.forEach((e) => {
+    e.locationName = e.country;
+  });
+  return countries;
+}
+
+export async function getOverviewData(counrty: string) {
+  let result, outdated;
+  if (counrty === "all") {
+    result = request(`all?yesterday=true`);
+    outdated = true;
+  } else {
+    result = await request(`countries/${counrty}`);
```

Comment:
My understanding of the `fetch` API is that it returns some `Promise<T>` -- but on line `19` you didn't `await` it, yet you did await it on line `22`? Which is the intention?

Reply:
Yep, typo... here. Although I don't even know why it works. `await` is needed.